about/src/index.html

```diff
@@ -11,8 +11,8 @@ <h1>${ABOUT_TR_DESCRIPTION}</h1>
   <p>Tag: <code>${DOCKER_TAG}</code></p>
   <p>Commit: <code>${SOURCE_COMMIT}</code></p>
   <ul>
-    <li><a href="${NGINX_VIRTUAL_HOST_SCHEME}${NGINX_VIRTUAL_HOST}${NGINX_LOCATION}">textrepo</a></li>
-    <li><a href="${NGINX_VIRTUAL_HOST_SCHEME}${NGINX_VIRTUAL_HOST}${NGINX_LOCATION}swagger/">swagger</a></li>
+    <li><a href="${NGINX_VIRTUAL_HOST_SCHEME}${NGINX_VIRTUAL_HOST}${NGINX_LOCATION}/">textrepo</a></li>
```

Comment:
hmm, dit kun je niet zomaar aanpassen zonder ook de env vars aan te passen, en volgens mij heb je dat niet gedaan?
Ik krijg nu bijvoorbeeld een 404 voor http://localhost:8080/textrepo//swagger/

Reply:
c85c8346: beide docker-compose.yml in examples/ aangepast en de wijziging in CHANGELOG.md gedocumenteerd